src/main/java/Command.java

```diff
@@ -0,0 +1,68 @@
+public class Command {
+    public static void findCommand(String taskInput){
+        String[] splitTaskInput = taskInput.toLowerCase().split(" ");
+        String[] taskDescription;
+        switch (splitTaskInput[0]){
+            case "done":
```

Comment:
Consider refactoring this switch case. The execution path taken when everything goes well should be clearer here. Consider wrapping each outcome into a method that best describes the outcome